### ISSUE TITLE
Bug: Interests in Profile Settings Need to Be Toggle Buttons

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#2148]
### Describe the bug

Currently you can't navigate by keyboard Interests on the profile settings page.

### Steps to reproduce

1. Go to your user settings page, https://app.opensauced.pizza/user/settings
2. Use the <kbd>TAB</kbd> key to tab through all the fields. Notice that it skips over the profile interests because they're divs.

![CleanShot 2023-11-14 at 22 35 53](https://github.com/open-sauced/app/assets/833231/58a3f3f0-0459-4d08-aff8-e1ac5ca673d6)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by nickytonline** [comment]
These should be toggle buttons, much like the ToggleSwitch component.

**Comment by nickytonline** [comment]
Relates to https://github.com/open-sauced/internal-team/issues/21
